src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/controller/SearchController.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.personrecord.api.controller
+
+import jakarta.validation.constraints.NotBlank
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.personrecord.api.controller.exceptions.PersonRecordNotFoundException
+import uk.gov.justice.digital.hmpps.personrecord.api.model.PersonIdentifierRecord
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.PersonEntity
+import uk.gov.justice.digital.hmpps.personrecord.jpa.repository.PersonRepository
+import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType
+
+@RestController
+@RequestMapping("/api")
```

Comment:
It's a minor point but I am not a big fan of splitting the request mapping over multiple anotations.
If we end up having 15 endpoints under `/api` say, the temptation (perhaps even requirement, not sure how the RequestMapping annotation works) would be that they all have to live in this controller, whether they are related or not.

Reply:
Fair point, updated 👍